src/recovery.rs

```diff
@@ -702,13 +741,18 @@ impl std::fmt::Debug for Recovery {
     }
 }
 
+#[derive(Clone)]
 pub struct Sent {
     pub pkt_num: u64,
 
     pub frames: Vec<frame::Frame>,
 
     pub time_sent: Instant,
 
+    pub time_acked: Option<Instant>,
```

Comment:
Looks there is `Instant` is not used in the code. Is it good enough to be `bool`?

Reply:
This is mostly to be consistent with the `time_lost`, though as per the PR description this can later be used for other things. There's not much harm in leaving it like this I think.